homeassistant/components/life360/button.py

```diff
@@ -0,0 +1,57 @@
+"""Support for Life360 buttons."""
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import Life360DataUpdateCoordinator
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Life360 buttons."""
+    coordinator: Life360DataUpdateCoordinator = hass.data[DOMAIN].coordinators[
+        config_entry.entry_id
+    ]
+    for member_id, member in coordinator.data.members.items():
+        async_add_entities(
+            [
+                Life360UpdateLocationButton(coordinator, member.circle_id, member_id),
+            ]
+        )
+
+
+class Life360UpdateLocationButton(
+    CoordinatorEntity[Life360DataUpdateCoordinator], ButtonEntity
+):
+    """Represent an Life360 Update Location button."""
+
+    _attr_unique_id: str
```

Comment:
Why do you need this line? Setting it on line 49 should be enough.

Reply:
Good point. Removed.